### PR TITLE
OKTA-628939: Apply hreflang alternates to H.O.C. index.html

### PR DESCRIPTION
**Pull request by paulwallace-okta**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
+<title>Okta Documentation</title>
```
Comment (emikhasyak-okta) [inline index.html]: 👍

File: index.html [hunk 2]
```diff
@@ -1,9 +1,13 @@
 <!DOCTYPE html>
 <html>
-<head>
-<title>Okta Documentation</title>
```
Comment (emikhasyak-okta) [inline index.html]: Is the title removed on purpose? It looked like a nice addition

**Review by emikhasyak-okta — APPROVED** [review]
LGTM! pls double check if you want to add the title or not and 🚢 afterward

File: index.html (hunk 2 above)
Comment (paulwallace-okta) [reply index.html]: I guess it can't hurt, added it back. thanks!

**Pull request merged**